solutions/observability/logs/filter-aggregate-logs.md

```diff
@@ -29,7 +29,9 @@ This guide shows you how to:
 ::::
 
 
-The examples on this page use the following ingest pipeline and index template, which you can set in **Developer Tools**. If you haven’t used ingest pipelines and index templates to parse your log data and extract structured fields yet, start with the [Parse and organize logs](/solutions/observability/logs/parse-route-logs.md) documentation.
+The examples on this page use the following ingest pipeline and index template. The pipeline and template need to be set before you create your data stream in the following steps. Set them in **Developer Tools**, which can be found by searching for `Developer Tools` in the [global search field](/explore-analyze/find-and-organize/find-apps-and-objects.md).
```

Comment:
This is a super clear para, but I'd remove the passive voice in the second sentence ("You need to set...")